Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp

```diff
@@ -629,9 +629,12 @@ void AXIsolatedTree::updateNodeProperties(AccessibilityObject& axObject, const A
         case AXProperty::BrailleRoleDescription:
             properties.append({ AXProperty::BrailleRoleDescription, axObject.brailleRoleDescription().isolatedCopy() });
             break;
-        case AXProperty::AXColumnIndex:
-            properties.append({ AXProperty::AXColumnIndex, axObject.axColumnIndex() });
+        case AXProperty::AXColumnIndex: {
+            std::optional<unsigned> columnIndex = axObject.axColumnIndex();
+            if (columnIndex)
+                properties.append({ AXProperty::AXColumnIndex, *columnIndex });
```

Comment:
I'm fine to leave as-is, but another way to write this would be:

```cpp
if (std::optional<unsigned> columnIndex = axObject.axColumnIndex())
    properties.append({ AXProperty::AXColumnIndex, *columnIndex });
```

Same comment for the other two changes in this file.

Reply:
Done, thanks!